prow/scripts/lib/testing-helpers.sh

```diff
@@ -229,3 +229,46 @@ function testing::remove_addons_if_necessary() {
       log::info "- Skipping removing ClusterAddonsConfiguration"
   fi
 }
+
+testing::runTestLogCollector(){
```

Comment:
Please change the function name to snake_case to preserve consistency

Reply:
Function names changed to snake_case